### PR TITLE
chore: use Ramp Network name

### PR DESCRIPTION
**Pull request by jakubsta**
Closes: #PRODUCT-23218

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Ramp Flutter
+# Ramp Network Flutter
 
 Official Flutter wrapper for Ramp
```
Comment (szymon-bladoszewski-ramp) [inline README.md:3]: Network missing

Comment (jakubsta) [inline README.md:3]:
```suggestion
Official Flutter wrapper for Ramp Network
```

File: ios/ramp_flutter.podspec [hunk 2]
```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
   s.name             = 'ramp_flutter'
   s.version          = '1.0.1'
-  s.summary          = 'Ramp iOS wrapper for Flutter.'
+  s.summary          = 'Ramp Network iOS wrapper for Flutter.'
   s.description      = <<-DESC
-  Ramp for Flutter is a simple wrapper for native iOS and Android Ramp SDKs.
+  Ramp Network for Flutter is a simple wrapper for native iOS and Android Ramp NetworknSDKs.
```
Comment (jakubsta) [inline ios/ramp_flutter.podspec:6]:
```suggestion
  Ramp Network for Flutter is a simple wrapper for native iOS and Android Ramp Networkn SDKs.
```

File: ios/ramp_flutter.podspec [hunk 3]
```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
   s.name             = 'ramp_flutter'
   s.version          = '1.0.1'
-  s.summary          = 'Ramp iOS wrapper for Flutter.'
+  s.summary          = 'Ramp Network iOS wrapper for Flutter.'
   s.description      = <<-DESC
-  Ramp for Flutter is a simple wrapper for native iOS and Android Ramp SDKs.
+  Ramp Network for Flutter is a simple wrapper for native iOS and Android Ramp Networkn SDKs.
```
Comment (jakubsta) [inline ios/ramp_flutter.podspec:6]:
```suggestion
  Ramp Network for Flutter is a simple wrapper for native iOS and Android Ramp Network SDKs.
```

**Review by szymon-bladoszewski-ramp — APPROVED** [review]

**Pull request merged**